### PR TITLE
Add section to README

### PR DESCRIPTION
**Pull request by krisdale**
#Description

Update README for consistency with other oneAPI repos. Minor edits for readability:

- Add Contribute section, linking to CONTRIBUTING
- Add link to LICENSE from License section
- Update issue list to bullets (not numbers) as it is not ordered
- Update issue list for consistency and readability
- Make section title capitalization consistent

## Type of change
Editorial/formatting change for README

# How Has This Been Tested?
NA

# Checklist for Moving samples:
NA

### CODE REVIEW

**Review by JoeOster — APPROVED** [review]

**Comment by JoeOster** [comment]
@krisdale please fix your conflicts

File: README.md [hunk 1]
```diff
@@ -80,15 +80,20 @@ be stable.
...
+## Known Issues or Limitations
 
 ### On Windows Platform
-1. If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
-2. To build samples on Windows, the required Windows SDK is ver. 10.0.17763.0. 
-3. Now you should be able to build the code sample.
-4. For beta, FPGA samples support Windows through FPGA-emulator.
-5. If you encounter a compilation error like below when building a sample program, one reason is that the directory path of the sample is too long; the work around is to move the sample to a directory like "c:\temp\sample_name".
-  * Error MSB6003 The specified task executable "dpcpp.exe" could not be run .......
+- If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
```
Comment (anjgola) [inline README.md:90]: @Jenn500  Is this infromation still accurate?

Comment (Jenn500) [reply README.md:90]: 1) there're compiler issues with some versions of the VS updates. those should be still needed. 
2) the Windows SDK requirement should not be needed. 
4) From the system requirement, still true. please check with Audrey to confirm.

Comment (akertesz) [reply README.md:90]: Yes, there was an obscure bug with a particular release of VS 2019. This is accurate.

File: README.md [hunk 2]
```diff
@@ -80,15 +80,20 @@ be stable.
...
 ### On Windows Platform
-1. If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
-2. To build samples on Windows, the required Windows SDK is ver. 10.0.17763.0. 
-3. Now you should be able to build the code sample.
-4. FPGA samples support Windows through FPGA-emulator.
-5. If you encounter a compilation error like below when building a sample program, one reason is that the directory path of the sample is too long; the work around is to move the sample to a directory like "c:\temp\sample_name".
-  * Error MSB6003 The specified task executable "dpcpp.exe" could not be run .......
+- If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
+- Windows SDK version 10.0.17763.0 is required to build samples on Windows.
+- FPGA samples support Windows through the FPGA-emulator.
```
Comment (akertesz) [inline README.md:92]: Please change this line to say:
"FPGA code sample Windows support is limited to the FPGA emulator and optimization reports. See any FPGA code sample for details."

**Review by JoeOster — APPROVED** [review]

**Pull request merged**